.pylintrc

```diff
@@ -5,4 +5,5 @@ disable=F0401,  # import loading
         E1101,  # function members
         R0801,  # duplicate code
         R0903,  # public methods
-        R0915   # too many statements
+        R0915,   # too many statements
+        R0914   # too many local variables
```

Comment:
Are we using an open source Google style guide for the linter?
If so, we should strongly consider disabling as few rules as possible, else it sorts of defeats the purpose (of the linter).
I'm not too sure about pylint specifically but there should also be a way to disable a certain rule _locally_, for a tiny chunk of code where the rule just doesn't make sense, while keeping it enabled elsewhere.
Thoughts?

Reply:
Filed a follow-up issue for this! https://github.com/GoogleCloudPlatform/bank-of-anthos/issues/465